lib/forklift/box_loader.rb

```diff
@@ -63,13 +63,15 @@ def process_versions(config, versions)
           katello_box = Marshal.load(
             Marshal.dump(config['boxes']["#{os}-katello-nightly"])
           )
-          katello_box['options'] ||= []
-          katello_box['options'] << " --version #{version}"
+          katello_box['ansible'] ||= {}
+          katello_box['ansible']['variables'] = { 'foreman_version' => version }
 
           capsule_box = Marshal.load(
             Marshal.dump(config['boxes']["#{os}-capsule-nightly"])
           )
+          capsule_box['ansible'] ||= {}
           capsule_box['ansible']['server'] = "#{os}-katello-#{katello_version}"
+          capsule_box['ansible']['variables'] = { 'foreman_version' => version }
```

Comment:
Hm, if dynamic boxes are going to supply variables, and a user creates a derived box in boxes.yaml, I think we need to merge them in Vagrantfile

https://github.com/ehelms/forklift/blob/08c266baf32d8dc228a9fd3d5adbc728cd49e11f/Vagrantfile#L68